content/client-server-api/modules/event_replacements.md

```diff
@@ -231,15 +244,14 @@ subsequent replacements are themselves redacted). Note that this behaviour is
 specific to the `m.replace` relationship. See also [redactions of edited
 events](#redactions-of-edited-events) below.
 
-##### Server-side replacement of content
-
-Whenever an `m.replace` is to be bundled with an event as above, the server
-should also modify the content of the original event according to the
-`m.new_content` of the most recent replacement event (determined as above).
+{{< changed-in v="1.7" >}} In previous versions of the specification, only the
+`event_id`, `origin_server_ts`, and `sender` of the replacement event were included
+in the aggregation. v1.7 extends the aggregation to the full event.
```

Comment:
Like the comment I've left on the paragraph below, the spec doesn't prefer to have inline changelogs like this (shouldn't say "In previous versions" if at all possible). Instead, this sort of text requirement can get rolled into a much earlier paragraph as a rule for implementations to follow - I'll leave a comment demonstrating that.

Reply:
resolving this thread in favour of https://github.com/matrix-org/matrix-spec/pull/1440#discussion_r1119582258